src/observations/units.jl

```diff
@@ -0,0 +1,147 @@
+@doc raw"""
+    julian2etsecs(jd)
+
+Convert `jd` julian days to ephemeris seconds since J2000.
+
+See also [`etsecs2julian`](@ref).
+"""
+function julian2etsecs(jd)
+    return (jd-JD_J2000)*daysec
+end
+
+@doc raw"""
+    etsecs2julian(et)
+
+Convert `et` ephemeris seconds since J2000 to julian days.
+
+See also [`julian2etsecs`](@ref).
+"""
+function etsecs2julian(et)
+    return JD_J2000 + et/daysec
+end
+
+@doc raw"""
+    datetime2et(x::DateTime)
+    datetime2et(x::T) where {T <: AbstractObservation}
+    
+Retun the TDB seconds past the J2000 epoch.
+
+See also [`SPICE.str2et`](@ref).
+"""
+function datetime2et(x::DateTime)
+    return str2et(string(x))
+end
+
+datetime2et(x::T) where {T <: AbstractObservation} = datetime2et(x.date)
+
+@doc raw"""
+    et_to_200X(et::T) where {T <: Number}
+
+Convert `et` ephemeris seconds since J2000 to years `200X`. 
+"""
+et_to_200X(et::T) where {T <: Number} = 2000 + et/daysec/yr
+
+@doc raw"""
+    days_to_200X(jd::T) where {T <: Number}
+
+Convert `jd` julian days since J2000 to years `200X`. 
+"""
+days_to_200X(jd::T) where {T <: Number} = 2000 + jd/yr
+
+@doc raw"""
+    datetime_to_200X(x::DateTime)
+
+Convert `x` to years `200X`. 
+"""
+datetime_to_200X(x::DateTime) = et_to_200X(datetime2et(x))
+
+@doc raw"""
+    datetime2days(x::DateTime)
+
+Convert `x` to julian days since 2000. 
+"""
+datetime2days(x::DateTime) = datetime2julian(x) - JD_J2000
+
+@doc raw"""
+    days2datetime(x::DateTime)
+
+Convert `jd` julian days since 2000 to `DateTime`. 
+"""
+days2datetime(jd::T) where {T <: Number} = julian2datetime(jd + JD_J2000)
```

Comment:
```suggestion
@doc raw"""
    days2datetime(d::T) where {T <: Number}

Convert `d` days since J2000 to `DateTime`. 
"""
days2datetime(jd::T) where {T <: Number} = julian2datetime(jd + JD_J2000)
```

Reply:
Done.